modules/cassandra/store/pom.xml

```diff
@@ -140,6 +140,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Mockito dependency should be specified in the "dependencyManagement" section of parent POM. Such way you don't need to specify <version> tag in all the modules which are going to reuse the dependecy - it will be automatically resolved from parent POM.

Reply:
added dependencyManagement to the parent POM, such that the child POM doesn't need to specify the version.

However, unless we add it to the parent as a 'dependency', then the child pom still needs to add it as a dependency.